src/asm/whitespace.c

```diff
@@ -87,11 +79,13 @@ void	whitespace_trim(char *str)
 		}
 		i++;
 	}
+	return ;
 }
 
 void	process_whitespace(char *str)
 {
 	whitespace_transform(str);
 	whitespace_consolidate(str);
 	whitespace_trim(str);
+	return ;
```

Comment:
Honestly, these returns are useless.  They make the function semantically correct, but it's really un-necessary, but don't worry about removing these.